Makefile

```diff
@@ -286,12 +286,19 @@ MINISHIFT_URL = http://$(MINISHIFT_IP)
 # make sure you have a entry in /etc/hosts for "minishift.local MINISHIFT_IP"
 MINISHIFT_HOSTS_ENTRY = http://minishift.local
 
+# Setup AUTH image URL, use default image path and default tag if not provided
+AUTH_IMAGE_DEFAULT=docker.io/fabric8/fabric8-auth
+AUTH_IMAGE_TAG ?= latest
```

Comment:
@kwk now using this assignment. thanks!

Reply:
@pranavgore09 Yeah, overall this looks much cleaner! Good job!